types.go

```diff
@@ -14,3 +14,32 @@ func GoString(s unsafe.Pointer) string {
 
 	return C.GoStringN((*C.char)(unsafe.Pointer(&zendStr.val)), C.int(zendStr.len))
 }
```

Comment:
Actually, I was mistaken. Unlike what I wrote in the docs `GoString` does a copy, so `GoStringCopy` can be removed.